internal/worker/sandbox_options.go

```diff
@@ -2,20 +2,24 @@ package worker
 
 import (
 	"github.com/ossf/package-analysis/internal/sandbox"
+	"github.com/ossf/package-analysis/pkg/api/pkgecosystem"
 )
 
 // StaticSandboxOptions provides a set of sandbox options necessary to run the
 // static analysis sandboxes.
 func StaticSandboxOptions() []sandbox.Option {
 	return []sandbox.Option{
+		sandbox.Image(defaultStaticAnalysisImage),
 		sandbox.EchoStdErr(),
 	}
 }
 
 // DynamicSandboxOptions provides a set of sandbox options necessary to run
 // dynamic analysis sandboxes.
-func DynamicSandboxOptions() []sandbox.Option {
+func DynamicSandboxOptions(ecosystem pkgecosystem.Ecosystem) []sandbox.Option {
 	return []sandbox.Option{
+		sandbox.Image(defaultDynamicAnalysisImage),
+		sandbox.Command(defaultDynamicAnalysisCommand[ecosystem]),
```

Comment:
I think it might be cleaner to pass the command in with `Run` maybe?

It avoids ambiguity, and makes it easier to do general workloads in the sandbox.